### PR TITLE
Ensure loadscope and loadfile work when the first scope has one work unit

### LINKED ISSUE #257: dist=loadscope scheduling hangs on test class with only one test
With pytest 3.2.3, when I tried using --dist=loadscope with two ssh nodes, the test framework simply hung without even trying to run any tests.  The same environment works fine with the other schedulers.

Part of the traceback (seems exactly the same as #230 ):
        while not self.session_finished:
>           self.loop_once()

The hang seems to occur when running a test class that has only one test function.
I eventually tracked down a simple change that seems to fix it: on the following line, expect only one item, not two:

https://github.com/pytest-dev/pytest-xdist/blob/master/xdist/remote.py#L62

I can't tell what the change might break, though.

### PR DESCRIPTION
**Pull request by lmtierney**
Fixes #257 

- [X] Make sure to include reasonable tests for your change if necessary

- [X] We use [towncrier](https://pypi.python.org/pypi/towncrier) for changelog management.

### CODE REVIEW

**Comment by mikeage** [comment]
Hi, sorry to be "that guy", but is there an ETA for merging this? I too have hit this bug, and while the solution here is working great, it would be nice to go back to an official pytest-xdist release and not have to pull from github. Thanks!

**Comment by nicoddemus** [comment]
Hi @mikeage I particularly have no problems with a gentle ping, thanks!

I will review this later and we can have a release early next week I believe. 👍

File: xdist/scheduler/loadscope.py [hunk 1]
```diff
@@ -380,6 +380,10 @@ def schedule(self):
         for node in self.nodes:
             self._assign_work_unit(node)
 
+        # Ensure nodes start with at least two work units if possible (#277)
+        for node in self.nodes:
```
Comment (RonnyPfannschmidt) [inline xdist/scheduler/loadscope.py]: while this looks correct it also may affect shutdown behaviour, as such i'd like to see how it acts in the wild

**Comment by nicoddemus** [comment]
`1.22.1` is out, thanks again @lmtierney for the PR!

**Pull request merged**